drivers/bridge/bridge.go

```diff
@@ -46,6 +42,8 @@ type Configuration struct {
 	EnableICC             bool
 	EnableIPForwarding    bool
 	AllowNonDefaultBridge bool
+	// Driver components
+	PortAllocator *portallocator.PortAllocator
```

Comment:
I get your point, but I am worried about making the assumption that portallocator is going to be only used for tracking host ports. It may not be the only case in future.

I see portallocator as a simple ip:proto:port database which could be used on the default as well as on other net namespaces.